functions/models/priceData.ts

```diff
@@ -0,0 +1,15 @@
+export class PriceData {
```

Comment:
Maybe include a 'last checked' timestamp? What if you end up wanting to include price data from multiple sources?